src/til/ut_til/EnumSetTests.cpp

```diff
@@ -101,15 +101,13 @@ class EnumSetTests
 
         Log::Comment(L"Start with bits 0, 2, and 4 set");
         til::enumset<Flags> flags{ Flags::Zero, Flags::Two, Flags::Four };
-        VERIFY_ARE_EQUAL(0b10101u, flags.to_ulong());
+        VERIFY_ARE_EQUAL(0b10101u, flags.bits());
 
-        Log::Comment(L"Test bits 1 and 2 with the test method");
-        VERIFY_IS_FALSE(flags.test(Flags::One));
-        VERIFY_IS_TRUE(flags.test(Flags::Two));
-
-        Log::Comment(L"Test bit 3 and 4 with the array operator");
-        VERIFY_IS_FALSE(flags[Flags::Three]);
-        VERIFY_IS_TRUE(flags[Flags::Four]);
+        Log::Comment(L"Test bits with the any method");
+        VERIFY_IS_FALSE(flags.any(Flags::One));
+        VERIFY_IS_TRUE(flags.any(Flags::Two));
+        VERIFY_IS_FALSE(flags.any(Flags::Three));
+        VERIFY_IS_TRUE(flags.any(Flags::Four));

```

Comment:
The point of these test cases were to validate the `test` method and the array ops. The array ops aren't used anymore, but we still need coverage for the `test` method, so maybe just make them all `test` calls. The `any` method is already covered in the test cases further down.

Reply:
Ah I forgot to revert those tests...